### PR TITLE
[#1693] Fix limit order SDK changes

### PR DESCRIPTION
**Pull request by wixzi**
# Summary

#1693 

Screenshot 2023-01-22 at 16 40 20

Now limit order `Maximum possible amount ` is showing after reducing the fees. Since the new CoW SDK change we don't want to pass the fees. 
So its a better experience for the users.
API for Cancel an open order has changed in the new SDK and need to make those changes to work.

  # To Test

- Should be able to place a limit order with the maximum available amount
- Should be able to cancel and open limit order.

### CODE REVIEW

File: src/pages/Swap/LimitOrderBox/components/LimitOrderForm/LimitOrderForm.tsx [hunk 1]
```diff
@@ -202,8 +202,8 @@ export function LimitOrderForm({ account, provider, chainId }: LimitOrderFormPro
 
       setMarketPriceInterval(refetchMarketPrice)
     } else {
-      clearInterval(marketPriceInterval)
       setMarketPriceInterval(undefined)
+      clearInterval(marketPriceInterval)
```
Comment (berteotti) [inline src/pages/Swap/LimitOrderBox/components/LimitOrderForm/LimitOrderForm.tsx:206]: Is there any difference in this order?

**Review by berteotti — APPROVED** [review]
Code looks good to me

**Comment by MilanVojnovic95** [comment]
@wixzi 
Tested and I can see a notification but it does not appear that the fees are reduced 🤔 

image

**Comment by wixzi** [comment]
> @wixzi Tested and I can see a notification but it does not appear that the fees are reduced 🤔
> 
> image
@MilanVojnovic95 
Yes, that's the change. We don't want to reduce the fees for the new SDK. The fee is auto-reduced from the sell token when the transaction is executed. So we pass fee as 0. So the max order you can place is the max available amount for you.

File: src/pages/Swap/LimitOrderBox/components/LimitOrderForm/LimitOrderForm.tsx (hunk 1 above)
Comment (wixzi) [reply src/pages/Swap/LimitOrderBox/components/LimitOrderForm/LimitOrderForm.tsx:206]: No :)

**Comment by MilanVojnovic95** [comment]
> > @wixzi Tested and I can see a notification but it does not appear that the fees are reduced 🤔
> > image
> > @MilanVojnovic95
> > Yes, that's the change. We don't want to reduce the fees for the new SDK. The fee is auto-reduced from the sell token when the transaction is executed. So we pass fee as 0. So the max order you can place is the max available amount for you.

Ok, got it wrong, then everything seems to be ok 👍

**Review by MilanVojnovic95 — APPROVED** [review]

**Pull request merged**